ocs_ci/ocs/ocs_upgrade.py

```diff
@@ -224,7 +224,7 @@ def load_version_config_file(self, upgrade_version):
         version_change = self.get_parsed_versions()[1] > self.get_parsed_versions()[0]
         if version_change:
             version_config_file = os.path.join(
-                constants.CONF_DIR, "ocs_version", f"ocs-{upgrade_version}.yaml"
+                constants.OCS_VERSION_CONF_DIR, "ocs_version", f"ocs-{upgrade_version}.yaml"
```

Comment:
constants.OCS_VERSION_CONF_DIR is already pointing to the path of ocs_version, so you do not need to joined it, just use :

`version_config_file = os.path.join(constants.OCS_VERSION_CONF_DIR, f"ocs-{upgrade_version}.yaml")`

Reply:
Thanks! didn't see that